app/_src/networking/transparent-proxy/universal.md

```diff
@@ -0,0 +1,468 @@
+{% enddanger %}
+
+```sh
+iptables --table nat --flush         # Flush all rules in the nat table (IPv4)
+ip6tables --table nat --flush        # Flush all rules in the nat table (IPv6)
+iptables --table nat --delete-chain  # Delete all custom chains in the nat table (IPv4)
+ip6tables --table nat --delete-chain # Delete all custom chains in the nat table (IPv6)
+
+# The raw table contains rules for DNS traffic redirection
+iptables --table raw --flush         # Flush all rules in the raw table (IPv4)
+ip6tables --table raw --flush        # Flush all rules in the raw table (IPv6)
+
+# The mangle table contains rules to drop invalid packets
+iptables --table mangle --flush      # Flush all rules in the mangle table (IPv4)
+ip6tables --table mangle --flush     # Flush all rules in the mangle table (IPv6)
+```
+
+### Step 2: Install new transparent proxy
+
+After clearing the `iptables` rules (if necessary), reinstall the transparent proxy by running:
+
+```sh
+kumactl install transparent-proxy [...]
+```
+
+This command will install the new version of the transparent proxy with the specified configuration. Adjust the flags as needed to suit your environment.
+
+## Configuration
+
+The default configuration works well for most scenarios, but there are cases where adjustments are needed.
+
+{{ Kuma }} uses a unified configuration structure for transparent proxy across all components. For a detailed breakdown of this structure, including examples, expected formats, and variations between configuration methods, refer to the [Transparent Proxy Configuration reference]({{ docs }}/reference/transparent-proxy-configuration/).
+
+In Universal mode, {{ Kuma }} there are three methods to adjust the configuration. Each can be used on its own or combined with others if needed.
+
+{% warning %}
+It’s best to stick to one method whenever possible. Using more than one can make things more complicated and harder to troubleshoot, as it may not be clear where each setting comes from. If you need to combine methods, check the [**Order of Precedence**](#order-of-precedence) section to see what the final configuration will look like based on the priority of each setting.
+{% endwarning %}
+
+<!-- vale Google.Headings = NO -->
+### YAML / JSON
+<!-- vale Google.Headings = YES -->
+
+You can provide the configuration in either `YAML` or `JSON` format by using the `--config` or `--config-file` flags.
+
+{% tip %}
+For the configuration schema in YAML format, refer to the [Schema]({{ docs }}/reference/transparent-proxy-configuration/#schema) section in the [Transparent Proxy Configuration Reference]({{ docs }}/reference/transparent-proxy-configuration/).
+{% endtip %}
+
+{% tip %}
+For simplicity, the following examples use YAML format, but you can easily convert them to JSON if preferred. Both formats work exactly the same, so feel free to choose the one that best suits your needs.
```

Comment:
There's no simpliicity here. yaml is a superset of json so yaml parsers can always parse json...